packages/version/src/__tests__/version-conventional-commits.spec.ts

```diff
@@ -228,7 +228,7 @@ describe("--conventional-commits", () => {
 
       const cwd = await initFixture("normal");
 
-      await new VersionCommand(createArgv(cwd, "--conventional-commits", "--conventional-prerelease"));
+      await new VersionCommand(createArgv(cwd, "--conventional-commits", "--conventional-prerelease", "--no-package-lockfile-only"));
```

Comment:
Why is it necessary to add those flags to the tests?

Reply:
because making my final decision I had switched it to the new flag and had to adjust a few unit tests in the process. I could remove them back now though or just leave them there

Reply:
Rmove them because it adds bloat to the unit test.

Reply:
fixed